content/4.validators/2.becoming-a-validator/8.chain-upgrade.md

```diff
@@ -93,7 +91,7 @@ sudo systemctl stop archwayd
 ```
 ::
 
-Replace the old binary: Replace the old binary with the new one:
```

Comment:
why replace the old binary? While using cosmovisor there is a separate folder for each binary version